nakamoto/roster.json

```diff
@@ -70,7 +70,16 @@
     },
     {
       "name": "Felix Saiz",
-      "github": "fsaiz97"
+      "github": "fsaiz97",
+      "materials": [
+        {
+          "type": "cv"
+        },
+        {
+            "type": "video",
+            "url": "https://youtu.be/j1htjun3UaI"
```

Comment:
Can you please change "url": "https://www.youtube.com/watch?v=ac-jJPTaSeM" to "url": "ac-jJPTaSeM"